src/service.js

```diff
@@ -260,6 +279,19 @@ class Service {
 		return action;
 	}
 
+	/**
+	 * Retrieve list of active contexts for the service
+	 * @returns {Set}
+	 * @private
+	 * @memberof Service
+	 */
+	_getActiveContexts() {
+		if (_.isFunction(this.broker.ContextFactory.getActiveContexts)) {
```

Comment:
I don't understand this part... why don't you just call `this.activeContexts` like [here](https://github.com/rmccallum81/moleculer/blob/68d554c4b2cfdb166cb0dbd5e8509488c3366a36/src/context.js#L105)

Reply:
Sorry this is a leftover from previous implementation, I'll change it.